packages/bezier-react/src/components/Text/Text.tsx

```diff
@@ -60,7 +62,8 @@ export const Text = forwardRef<HTMLElement, TextProps>(function Text(props, forw
       styles[`typo-${typo}`],
       bold && styles.bold,
       italic && styles.italic,
-      truncated && styles.truncated,
+      truncated === true && styles.truncated,
+      isMultiLineTruncated && styles['multi-line-truncated'],
```

Comment:
```suggestion
      truncated === true
        ? styles.truncated
        : isMultiLineTruncated && styles['multi-line-truncated'],
```

`styles.truncated`, `styles['multi-line-truncated']` 두 클래스의 배타성이 잘 드러나면 더 좋을 거 같아요.